src/core/manager.c

```diff
@@ -2610,23 +2610,23 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         uint32_t id;
 
                         if (safe_atou32(val, &id) < 0)
-                                log_debug("Failed to parse current job id value %s", val);
```

Comment:
I agree with @keszybz . It appears that a deserialisation error indicates a bug, not some expected event.

By the way, I spent two hours trying to find out why `daemon-reexec` had frozen `systemd`. It would have helped a lot if I had seen something like `Failed to parse environment entry`.